eks/managing_user_access.md

```diff
@@ -0,0 +1,233 @@
+# Managing Access to your Kubernetes cluster
+The following document assumes you have used the terraform scripts found in this repository to create a Kubernetes cluster for your CircleCI server installation. Below we go through our recommended approach to providing and managing user access.
+
+
+## Adding Cluster Administrators
+Cluster admins are first registered via appending their IPs and IAM user details to the appropriate lists in your `terraform.tfvars` file.
+
+example:
+```
+allowed_cidr_blocks = ["user_ip/32","user1_ip/32"]
+k8s_administrators = [
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user ARN>"
+      username = "<username>"
+  },
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user1 ARN>"
+      username = "<user1name>"
+  }
+}
+```
+
+Upon `terrafrom apply`, these fields will whitelist your IPs, and add the list of users will be added to the system:masters group for full access to all resources in the cluster.
```

Comment:
This sentence doesn't seem right

Reply:
the line probably isn't necessary to i'll remove